### ISSUE TITLE
Run as a Service?

### ISSUE THREAD

**Issue by jeffrschneider** [diasks2/pragmatic_segmenter#54]
I'd like to use the segmenter with a large Java application and would prefer not to use JRuby, etc. I'd prefer to run the segmenter as a light-weight JSON service. Apologies, I don't know Ruby or I would read the code, but can you run this project as an always-on service?

**Comment by wildgarden** [comment]
@jeffrschneider Yes, indeed it is possible to run this project as a service - with slight modifications. However, it does not work out-of-the-box, you have to do is to wrap it in a service. We successfully use an existing  [.NET port](https://github.com/UglyToad/PragmaticSegmenterNet) of the project inside a dockerized ASP.CORE service which is called from Java. I'm sure, the same approach works with the original implementation.
